src/main/java/ai/tecton/client/request/GetFeaturesRequest.java

```diff
@@ -17,7 +17,7 @@ public class GetFeaturesRequest extends AbstractGetFeaturesRequest {
   static final String ENDPOINT = "/api/v1/feature-service/get-features";
   private final JsonAdapter<GetFeaturesRequestJson> jsonAdapter;
   private final GetFeaturesRequestData getFeaturesRequestData;
-  private final Moshi moshi = new Moshi.Builder().build();
+  private final Moshi moshi = new Moshi.Builder().add(SerializeNulls.JSON_ADAPTER_FACTORY).build();
```

Comment:
Curious is Moshi uses reflection in the hot code path or only once lazily and then other moshi interactinos are fast? I'm asking cause of performance requirements (and I'm not super familiar with moshi)

Reply:
AFAIK its lazy and the first use is costlier in terms of performance but rest of the uses are faster